src/NuGet.Indexing/Handlers/LatestListedHandler.cs

```diff
@@ -105,5 +111,11 @@ private static bool GetListed(Document document)
             string listed = document.Get("Listed");
             return (listed == null) ? false : listed.Equals("true", StringComparison.InvariantCultureIgnoreCase);
         }
+
+        private static bool GetSemver2(Document document)
```

Comment:
This will work with today's documents right? Meaning `document.Get("SemverLevel")` just returned null instead of throwing when this field does not exist.